seed/fieldtrials_testing_config_generator.py

```diff
@@ -144,7 +147,11 @@ def main():
         print("Seed data is invalid")
         return -1
     seed_message = serialize.make_variations_seed_message(seed_data)
-    target_unix_time = dateutil.parser.parse(args.target_date).timestamp()
+
+    date = datetime.strptime(args.target_date, '%Y-%m-%d %H:%M:%S %z')
+    target_unix_time = date.timestamp()
+
+    assert re.match(r'^\d+.\d+.\d+.\d+$', args.target_version)
```

Comment:
shouldn't this be `^\d+\.\d+\.\d+\.\d+$`?

Reply:
yep, thanks for the clarification.
The fix: https://github.com/brave/brave-variations/pull/854